autoconfigure/pom.xml

```diff
@@ -72,6 +78,13 @@
                     <excludeGroupIds>
                         io.zipkin.zipkin2,io.zipkin.reporter2,org.springframework.boot,org.springframework,com.fasterxml.jackson.core,com.google.auto.value,com.google.gson
                     </excludeGroupIds>
+                    <excludes>
+                        <!-- zipkin-server depends on armeria-->
+                        <exclude>
```

Comment:
are you depending on armeria here? if not, then you don't need to worry about this and it is a little distracting